packages/test-runner-module-mocking/src/moduleMockingPlugin.ts

```diff
@@ -37,15 +37,37 @@ export function moduleMockingPlugin(): Plugin {
           const namedExports = exports.map(e => e.n).filter(n => n !== 'default');
           const hasDefaultExport = exports.some(e => e.n === 'default');
 
-          body = `
+          body = generateInterceptedModuleBody(body, url, namedExports, hasDefaultExport);
```

Comment:
There is no real reason to extract this in a separate method. Can you revert this change?